autoPyTorch/pipeline/components/setup/traditional_ml/traditional_learner/utils.py

```diff
@@ -0,0 +1,13 @@
+AUTOPYTORCH_TO_CATBOOST_METRICS = {
```

Comment:
Can we make it by `Enum`?
By doing so, we cannot allow any assignments and we can avoid typos.

Reply:
sure